src/main-thread/commands/offscreen-polyfill-calls.ts

```diff
@@ -0,0 +1,49 @@
+import { CommandExecutor } from './interface';
+import { WorkerContext } from '../worker';
+import { OffscreenContextPolyfillMutationIndex } from '../../transfer/TransferrableMutation';
+import { NumericBoolean } from '../../utils';
+import { Strings } from '../strings';
+
+export function OffscreenPolyfillCallProcessor(strings: Strings, workerContext: WorkerContext): CommandExecutor {
+  return {
+    execute(mutations: Uint16Array, startPosition: number, target: RenderableElement): number {
+      const float32Needed = mutations[startPosition + OffscreenContextPolyfillMutationIndex.Float32Needed] === NumericBoolean.TRUE;
+      const mutationsArray = float32Needed ? new Float32Array(mutations.slice(startPosition).buffer) : mutations.slice(startPosition);
```

Comment:
This might throw in a few cases. Eg, if a `transfer` with an odd number of mutations comes after the polyfill call.